docs/content/docs/connectors/datastream/filesystem.md

```diff
@@ -792,6 +792,36 @@ class PersonVectorizer(schema: String) extends Vectorizer[Person](schema) {
 {{< /tab >}}
 {{< /tabs >}}
 
+For PyFlink users, `OrcBulkWriters.for_row_data_vectorization` could be used to create `BulkWriterFactory` to write `Row` records to files in Orc format.
+It should be noted that if the preceding operator of sink is an operator producing `RowData` records, e.g. CSV source, it needs to be converted to `Row` records before writing to sink.
```

Comment:
Maybe we need to give a some description to help pyflink users to understand `RowData`, which can be a doc link or something.

Reply:
I'll try to eliminate this in another PR.